src/env.h

```diff
@@ -510,6 +530,7 @@ class Environment {
   AsyncHooks async_hooks_;
   DomainFlag domain_flag_;
   TickInfo tick_info_;
+  ArrayBufferAllocator array_buffer_allocator_;
```

Comment:
Mmm.. but it will be allocated twice now? Once here, and once below when you do `new ArrayBufferAllocator`